src/sources/socket/udp.rs

```diff
@@ -199,14 +218,38 @@ pub(super) fn udp(
 
                                 for event in &mut events {
                                     if let Event::Log(ref mut log) = event {
-                                        log.try_insert(log_schema().source_type_key(), Bytes::from("socket"));
-                                        log.try_insert(log_schema().timestamp_key(), now);
-                                        log.try_insert(host_key.as_str(), address.ip().to_string());
+                    log_namespace.insert_standard_vector_source_metadata(
+                        log,
```

Comment:
formatting seems wack again? :thinking:

Reply:
Yep, same problem as the other instance.